conanfile.py

```diff
@@ -198,6 +198,8 @@ def system_requirements(self):
                 package_tool.install(package)
 
     def _copy_pkg_config(self, name):
+        if not name in self.deps_cpp_info.deps:
```

Comment:
Could you please explain this further?

Reply:
@ericLemanissier ^

Reply:
It could be a workaround for a bug in conan 1.28.0. I never tested without this change on conan 1.28.1. It's harmless anyway